services/integration-content-repository/app/api/swagger/swagger.json

```diff
@@ -81,798 +40,93 @@
         "summary": "Retrieve all flows associated with the current user from the repository",
         "description": "This will only return Flows that are either directly associated with the user, or that are associated with a tenant that the user is also a member of.",
         "operationId": "getFlows",
-        "responses": {
-          "200": {
-            "description": "A list of flows.",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/FlowResponse"
-                }
-              }
-            }
-          },
-          "404": {
-            "description": "No flows associated with the current user could be found in the repository."
-          }
-        }
-      },
-      "post": {
-        "x-swagger-router-controller": "flow",
-        "security": [
+        "parameters": [
           {
-            "Bearer": [
-
-            ]
-          }
-        ],
-        "tags": [
-          "flows"
-        ],
-        "summary": "Add a new flow to the repository",
-        "description": "The new Flow will automatically be associated with the current user through a \"user\"-type relationship, allowing editing and viewing by this user irrespective of tenancies.",
-        "operationId": "addFlow",
-        "responses": {
-          "200": {
-            "description": "The newly created Flow.",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/FlowResponse"
-                }
-              }
+          "name": "page[number]",
+          "in": "query",
+          "description": "Amount of flows per page returned. Default is 10.",
+          "required": false,
+          "schema": {
+            "type": "integer"
             }
           },
-          "409": {
-            "description": "Flow with this oihid already exists, oihid must be unique"
-          }
-        },
-        "requestBody": {
-          "content": {
-            "application/json": {
-              "schema": {
-                "$ref": "#/components/schemas/FlowRequest"
-              }
+          {
+          "name": "page[size]",
+          "in": "query",
+          "description": "Number of the page to be returned. Default is 1.",
+          "required": false,
```

Comment:
Should be `page[number]`.

See definition from [API Service Guidelines](https://github.com/openintegrationhub/openintegrationhub/blob/icr_new_flow_model/Guidelines/ServiceAPIGuidelines.md#pagination):
> page[number]: number of page the client wishes to receive. Corresponds to meta.page defined above. For example page[number]=2 tells the server to return the 2nd page.

Reply:
done